pkg/scaling/executor/scale_jobs.go

```diff
@@ -6,7 +6,6 @@ import (
 
 	"github.com/go-logr/logr"
 	batchv1 "k8s.io/api/batch/v1"
-	corev1 "k8s.io/api/core/v1"
```

Comment:
could we actually use 
`corev1 "k8s.io/api/core/v1"` and not `v1 "k8s.io/api/core/v1"` in this file?

Reply:
Yep, no problem! This should be fixed in the latest commit.